charts/background-worker/values.yaml

```diff
@@ -26,6 +26,7 @@ config:
   backendNotificationPusher:
     pushBackoffMinWait: 10000 # in microseconds, so 10ms
     pushBackoffMaxWait: 300000000 # microseconds, so 300s
+    remotesRefreshIntervalMs: 30000
```

Comment:
Adding `Ms` here has two problems:
1. Capital `M` means `Mega` not `micro`.
2. The things above do not have this suffix, so it is inconsistent.

Reply:
*sigh*

i really don't like the way we do config files.  very brittle and fragile at the same time.

i'll remove the Ms.